plugins/org.jkiss.dbeaver.ext.tidb/src/org/jkiss/dbeaver/ext/tidb/mysql/model/TiDBDialect.java

```diff
@@ -0,0 +1,68 @@
+package org.jkiss.dbeaver.ext.tidb.mysql.model;
+
+import java.util.Arrays;
+
+import org.jkiss.code.NotNull;
+import org.jkiss.dbeaver.ext.mysql.model.MySQLDialect;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCDatabaseMetaData;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCSession;
+import org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource;
+import org.jkiss.utils.ArrayUtils;
+
+public class TiDBDialect extends MySQLDialect {
```

Comment:
Please, add correct indentations and remove tabs from this class.

Reply:
Replaced by 4 spaces.